### PR TITLE
Update `History_of_osu!/osu!_wiki`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
[related discussion on discord](https://discord.com/channels/188630481301012481/218677502141399041/983741187041140796)

Proposal to update the article, so that the `outdated` tag may be removed.

The most important change is that I brought in recently added features, e.g. the lazer integration and footnote support. Besides that, I improved the readability and fixed a link which was not working anymore.

One thing that I'm unsure about is the link in line 18 (*green diff site*). For me, it is just a redirect to the new website, and does not show any temporary backup as described.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by TicClick** [comment]
further research shows that this was marked as `outdated` in [Dec 2019](https://github.com/ppy/osu-wiki/commit/cc7d660244df85d00e4a6a6d19691f53e8fa3b38#diff-e554b00d15254f2e08352e52458fa144d11b4f0e7d17b27951a689af323dfc9a), so you also need to mention things that took place since the last update (2018) to properly refresh the article

**Comment by The-Last-Cookie** [comment]
My contributions for the wiki began around October 2021, so I don't really have knowledge about the things that happened prior to that. There are circa [6000 PRs](https://github.com/ppy/osu-wiki/pulls?q=is%3Apr+is%3Aclosed+created%3A2018-01-01..2022-10-06+) between 2018 and today (2022-10-06), so I need some help in what information needs to be added in this article.

Otherwise, let me know if I should readd the `outdated` tag (and a comment of course).

File: wiki/History_of_osu!/osu!_wiki/en.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
----
-outdated: true
----
```
Comment (cl8n) [inline wiki/History_of_osu!/osu!_wiki/en.md:3]: maybe add `needs_cleanup` though cuz a lot of these entries seems kinda sloppily written and some exact dates are missing for whatever reason

**Comment by TicClick** [comment]
removing my request for review as I'm unlikely to get this done in near future. please go with the `needs_cleanup` suggestion (and the `outdated` tag can be removed, yeah)

**Review by cl8n — DISMISSED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**